src/pages/privacy-policy/privacy-policy.js

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import parse from 'html-react-parser';
+
+import { useStyles } from './privacy-policy.style';
+import { getBusinessPageByCode } from '../../redux/businessPages/businessPages.actions';
+
+const PrivacyPolicy = () => {
+  const dispatch = useDispatch();
+  const { privacyPolicyPage, language } = useSelector(
+    ({ BusinessPages, Language }) => ({
+      privacyPolicyPage: BusinessPages.pages.privacyPolicy,
+      language: Language.language
+    })
+  );
+
+  useEffect(() => {
+    window.scrollTo(0, 0);
+    dispatch(getBusinessPageByCode('privacy-policy'));
+  }, [dispatch]);
+
+  const privacyPolicyText = Object.keys(privacyPolicyPage).length
```

Comment:
Do we really need this check? I think business pages should always be translated, it should be ensured on the backend and admin side. Here we should not care about it